src/main/java/me/desht/pneumaticcraft/client/gui/GuiProgrammer.java

```diff
@@ -209,13 +209,13 @@ public void initGui() {
         buttonList.add(convertToRelativeButton);
 
         String containerName = I18n.format(te.getName() + ".name");
-        addLabel(containerName, guiLeft + 7, guiTop + 5, 0xFFFFFFFF);
+        addLabel(containerName, guiLeft + 7, guiTop + 5, 0xFF404040);
```

Comment:
Does that contrast well enough with the dark grey background of the programmer?  I'd assumed white was a deliberate choice here for better contrast...